pom.xml

```diff
@@ -117,6 +117,7 @@
         <logback.version>1.2.10</logback.version>
         <junit.version>4.13.2</junit.version>
         <je.version>7.3.7</je.version>
+        <plugin.surefire.version>2.21.0</plugin.surefire.version>
```

Comment:
Why not use 2.22.x ?

Reply:
The manager module above 2.22.x will not run test files
<img width="774" alt="image" src="https://user-images.githubusercontent.com/20356765/156739845-fe28c57c-6150-447d-8b31-f64e062ce09f.png">
<img width="782" alt="image" src="https://user-images.githubusercontent.com/20356765/156740110-c9b76a05-4ed1-4a54-8700-92d909045927.png">